packages/rich-text/src/parser/MdParser.js

```diff
@@ -88,22 +88,21 @@ const parse = code => (state, silent) => {
     return false;
 };
 
-
 class MdParser {
     constructor() {
         // disable all rules, enable autolink for URLs and email addresses
-        md = new MarkdownIt("zero", { linkify: true });
+        md = new MarkdownIt('zero', { linkify: true });
 
         // *bold* -> <strong>bold</strong>
-        md.inline.ruler.push("strong", parse(codes.bold.name));
+        md.inline.ruler.push('strong', parse(codes.bold.name));
 
         // _italic_ -> <em>italic</em>
-        md.inline.ruler.push("italic", parse(codes.italic.name));
+        md.inline.ruler.push('italic', parse(codes.italic.name));
 
         // :-) :) :-( :( :+1 :-1 -> <span>[unicode]</span>
-        md.inline.ruler.push("emoji", parse(codes.emoji.name));
+        md.inline.ruler.push('emoji', parse(codes.emoji.name));
 
-        md.enable(["linkify", "strong", "italic", "emoji"]);
+        md.enable(['link', 'linkify', 'strong', 'italic', 'emoji']);
```

Comment:
I think the addition of `'link'` here is the only change that is more than just formatting. I am approving on the basis of this assumption. If there were more "real" changes in this PR, please let me know and I'll have another look.

Reply:
Yes I think that's correct, the commits are also split up accordingly. LGTM

Reply:
Yes, I used a separate commits:
*  for the formatting: https://github.com/dhis2/d2-ui/pull/730/commits/0c793f1e83e51121911a781a5c5d6bfe607ba2ab
* for the actual fix: https://github.com/dhis2/d2-ui/pull/730/commits/45946bbc8c39d89e7d5635fb035067a5ea0c0134